prac_05/hex_colours.py

```diff
@@ -0,0 +1,22 @@
+
+"""
+CP1404/CP5632 Practical
+hex colours in a dictionary
+"""
+
+CODE_TO_NAME={"Absolute Zero": "#0048ba","Acid Green" : "#b0bf1a", "Alice Blue":"#f0f8ff","Baby Pink" : "#f4c2c2","Barn Red" : "#7c0a02","Beaver" : "#9f8170", "Bitter Lime" : "#bfff00","Blond" : "#faf0be","buff" : "#f0dc82","Camel" : "#c19a6b"}
+
```

Comment:
"buff" should be "Buff" ^^

Reply:
I will fix that .
Thanks you.

Reply:
Changes applied to the comment, great job! ^^